test/beast/http/fields.cpp

```diff
@@ -988,9 +988,25 @@ class fields_test : public beast::unit_test::suite
         BEAST_EXPECT(res[field::transfer_encoding] == "chunked, foo");
     }
 
+    void
+    testIssue1828()
+    {
+        beast::http::fields req;
+        req.insert("abc", "1");
+        req.insert("abc", "2");
+        req.insert("abc", "3");
+        BEAST_EXPECT(req.count("abc") == 3);
+        auto iter = req.find("abc");
+        BEAST_EXPECT(iter->value() == "1");
+        req.insert("abc", "4");
+        req.erase(iter);
+        BEAST_EXPECT(req.count("abc") == 3);
+    }
+
     void
     run() override
     {
+        testIssue1828();
```

Comment:
The order of calls should match the order of function declarations/definitions. Since you put `testIssue1828()` after all the other functions (which is the right thing to do), it should be called after all the other functions.

Reply:
done